test/test-framework-spec.md

```diff
@@ -154,6 +154,20 @@ The following are the list of available assertions available in the test framewo
    <td>Checks if the specified value is not equal to the expected value.
    </td>
   </tr>
+  <tr>
+     <td>@test:assertExactEquals
+     </td>
+     <td>Checks if the specified value is exactly equal to the expected value
+     i.e. both refer to the same entity.
+     </td>
+    </tr>
+    <tr>
+     <td>@test:assertExactNotEquals
```

Comment:
shouldn't this be assertNotExactEquals ?

Reply:
Yes, you are right.